index.html

```diff
@@ -25,6 +29,27 @@ <h1 id="logo">Aziz Mejri</h1>
           </li>
         </ul>
       </nav>
+      <!--mobile links-->
+         <div class="mobile-links" id="mobile-links">
+          <div class="link-container">
+            <div class="close-toggle" id="close-toggle">
+              <i class="fa fa-times" aria-hidden="true"></i>
+            </div>
+            <div class="ul">
+              <ul class="link mb-ul">
+                <li class="nav-item">
+                  <a class="is-link mb-link" href="#projects">Portfolio</a>
+                </li>
+                <li class="nav-item">
+                  <a class="is-link mb-link" href="#about-me">About</a>
+                </li>
+                <li class="nav-item">
+                  <a class="is-link mb-link" href="#mcontact">Contact</a>
```

Comment:
Please make sure that the `href` you are using matches the `id` of the section you're trying to reference as the last code reviewer suggested. :eyes: